mmdet/models/necks/fpn.py

```diff
@@ -34,6 +35,14 @@ class FPN(nn.Module):
         norm_cfg (dict): Config dict for normalization layer. Default: None.
         act_cfg (str): Config dict for activation layer in ConvModule.
             Default: None.
+        upsample_cfg (dict): Config dict for interpolate layer.
+            Default: `dict(mode='nearest')`
+        extra_convs_source (str, optional): The source feature map of the extra
+            convs. `None` means it is specified by `extra_convs_on_inputs`.
+            Otherwise, only the following modes are allowed
+             'inputs': Last feature map of neck inputs (i.e. backbone feature).
+             'laterals':  Last feature map after lateral convs
+             'outputs': The last output feature map after fpn convs
```

Comment:
I suggest using `input`, `lateral`, `output`.

Reply:
Removed extra_convs_source according to the last comment.